corehq/apps/receiverwrapper/util.py

```diff
@@ -79,6 +76,23 @@ def get_version_from_build_id(domain, build_id):
         return build.version
 
 
+@quickcache(['domain', 'build_id', 'property_name'], timeout=24 * 60 * 60)
+def get_profile_property_from_build_id(domain, build_id, property_name):
+    if not build_id:
+        return None
+
+    try:
+        build = get_app(domain, build_id)
```

Comment:
is get `get_app` cached? I imagine it gets called a lot in this part of the code

Reply:
Switched to `get_app_cached` in 453c960ec4b